core/base/planarGraphLayout/MergeTreeVisualization.h

```diff
@@ -0,0 +1,929 @@
+/// \ingroup base
+/// \class MergeTreeVisualization
+/// \author Mathieu Pont (mathieu.pont@lip6.fr)
+/// \date 2021.
+///
+/// Visualization module for merge trees.
+///
+
+#ifndef _MERGETREEVISUALIZATION_H
+#define _MERGETREEVISUALIZATION_H
+
+#pragma once
+
+#include <FTMTree.h>
+
+#include <stack>
+
+using namespace ttk;
+using namespace ftm;
+
+class MergeTreeVisualization : virtual public Debug {
+protected:
+  // Visualization parameters
+  bool PlanarLayout = false;
```

Comment:
by convention, the internal variables of a class in ttk's base code layer start with a small letter and finish with a "_" (to clarify it's an internal variable). that be great if you could `sed` all your classes in that perspective

Reply:
Ok thanks! I will apply this in the base layer classes in the next commit